Package@swift-5.8.swift

```diff
@@ -0,0 +1,94 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+import Foundation
+
+let package = Package(
+    name: "Scipio",
+    platforms: [
+        .macOS(.v12),
+    ],
+    products: [
+        .executable(name: "scipio",
+                    targets: ["scipio"]),
+        .library(
+            name: "ScipioKit",
+            targets: ["ScipioKit"]),
+        .library(
+            name: "ScipioS3Storage",
+            targets: ["ScipioS3Storage"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/giginet/swift-package-manager.git",
+                 revision: "99575e7a3eb8e66112ec45e2f11ef23a1828b110"),
```

Comment:
I upgrade the fork to the commit https://github.com/giginet/swift-package-manager/commit/99575e7a3eb8e66112ec45e2f11ef23a1828b110

SwiftPM refers HEAD of swift-driver and swift-support-tools-core.
Current HEAD of them only supports above macOS 12.0. But old SwiftPM fork still supports legacy macOS.
This causes a resolving error. So I bump up supported version on our fork.

This fork should be used for legacy Xcode users (14.3). So it will be removed after dropping 14 support.